sdk/src/runtime/requestInfo/types.ts

```diff
@@ -2,11 +2,17 @@ import { RwContext } from "../lib/router";
 
 export interface DefaultAppContext {}
 
+// NOTE: (peterp, 2025-08-01):
+// I think we should rename this to something other than request-info.
+// Since it's both the request and the response.
+// HttpMessages?
```

Comment:
It took my brain some time, but I now think of this as a live global interface for state associated with the current request. Since the response is also associated with the original request, I don't see a problem with 'requestInfo'.